packages/graphql-yoga/src/plugins/request-validation/use-http-validation-error.spec.ts

```diff
@@ -0,0 +1,52 @@
+import { createSchema } from '../../schema.js'
+import { createYoga } from '../../server.js'
+
+describe('useHTTPValidationError', () => {
+  describe('when doing a request with an invalid query', () => {
+    it('does not overrite status code set on custom plugin', async () => {
```

Comment:
Does this fail on the main branch?

Reply:
The use case is to throw status code 400 when there is a validation error, which is not happening in the main branch. I still do not follow why is that but I found this way to "workaround" that

Reply:
The idea is to allow users to modify http status code in the response by using plugins, this is actually not allowed since the `useHTTPValidationError` is added at the end of the plugins list and is overriding everything set before.
I found this issue while migrating from express-graphql, we actually need a way to differentiate from server errors vs validation errors, and this was the way we found to satisfy that requirement